easybuild/easyconfigs/x/XZ/XZ-5.2.2-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'XZ'
+version = '5.2.2'
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://tukaani.org/xz/']
+
+builddependencies = [
+    ('Autotools', '20150215'),
+    ('gettext', '0.19.8', '', True),
```

Comment:
note: the `cannot find 'ld'` issue only happens on systems where `ld.gold` isn't available system-wide...